app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/roadbook/RoadBookFragmentTest.kt

```diff
@@ -1051,6 +1167,86 @@ class RoadBookFragmentTest {
             .substringBeforeLast(":")
     }
 
+    private fun updateTimestampOfRecord(index: Int) {
+        if (index < 0 || index >= DestinationRecords.RECORDS.size) {
+            throw IllegalArgumentException("Index out of bounds")
+        }
+        onView(withText(DestinationRecords.RECORDS[index].destID)).check(matches(isDisplayed()))
+        swipeRightTheRecordAt(index)
+        onView(withId(R.id.editTextTimestamp)).perform(click())
+        onView(withText(timePickerUpdateBLabel)).perform(click())
+        onView(withText(R.string.edit_dialog_update_b)).perform(click())
+
+    }
+
+    private fun getRbViewModel(): RoadBookViewModel? {
```

Comment:
Prefer using the less possible that getter, it was designed for the Drag & Drop that I could not trigger by simulated touch moves.
However for me the way of testing matters less, and you can let that for the merge.